src/OnlineRetailPortal.Core/Models/Price.cs

```diff
@@ -8,6 +8,6 @@ public class Price
     {
         public double Amount { get; set; }
         public bool IsNegotiable { get; set; }
-        public Currency Currency { get; }
+        public string Currency { get; set; }
```

Comment:
Add a note that this is being handled with other pr .. 
aditi is working on these changes for bundling currency and amount

Reply:
You mean to say there should be type Money which contains Amount and Currency ??